docs/docs/index.md

```diff
@@ -46,10 +44,20 @@ time.sleep(2)
 print(job.result)   # => 889
 ```
 
+If your redis server is not local, you can define this in a config object:
+
+```python
+from rq import Queue, Config
+from redis import Redis
+
+config = Config(connection=Redis(REDIS_IP_OR_DOMAIN))
+q = Queue(config=config)
+```
+
 If you want to put the work on a specific queue, simply specify its name:
 
 ```python
-q = Queue('low', connection=redis_conn)
+q = Queue('low')
```

Comment:
I don't think we can get rid of `connection` for these reasons:
1. Storage must be explicitly defined
2. I want users to be able to easily use RQ with default options without having to worry about config

So the minimum way to define a queue would be `queue = Queue(connection=redis)`.